bsp_diff/caas/device/intel/civ/host/vm-manager/0026-Update-civ-1.ini-file-as-per-setup_civ.patch

```diff
@@ -0,0 +1,129 @@
+From 3b154924967765cc620361586dbf3865a79d269f Mon Sep 17 00:00:00 2001
+From: "Kothapeta, BikshapathiX" <bikshapathix.kothapeta@intel.com>
+Date: Mon, 17 Apr 2023 19:20:08 +0530
+Subject: [PATCH] Update civ-1.ini file as per setup_civ
+
+adb port net attributes are missed as setup-civ-ini.sh script updating.
```

Comment:
Any idea why this sample ini file is different from Android 12?
Can we get them to update the horizontal?

Reply:
Its already similar to Horizontal as rebased from Horizontal and setup_civ_ini.sh file script is updating based on this sample file, adb emulation ports are not  added in civ-sriov.ini file for this reason I have to made this change as per the Prashanth Suggestion.